### ISSUE TITLE
Lieux-dits sur polygones tels que "landuse" mais non rapprochés

### ISSUE THREAD

**Issue by acubens71** [osm-fr/bano#118]
(re-)Bonjour,
Certains lieux-dits désignant des entités tels que des bois, des étangs, des champs, des prés, ... il est logique que le nom du lieu-dit figure sur le polygone qui lui est associé (water, forest, meadow, farmland, ...), si le périmètre du polygone OSM concorde bien avec le périmètre du leu-dit dans le cadastre.
Exemple ici à Sornay (71) où ça marche bien : http://layers.openstreetmap.fr/?zoom=16&lat=46.60603&lon=5.18728&layers=0B000FFFFFFFFFFFFFFFFFFFFFT

Problème : ces noms ne sont pas rapprochés dans BANO. Serait-ce donc possible de permettre ce genre de rapprochements ?

Romuald71

**Comment by vdct** [comment]
Oui, ce manque devient plus flagrant depuis l'arrivée des lieux-dits. Il va falloir ouvrir le rapprochement, en considérant dans un premier temps les landuse=* + name=*. On verra à l'usage.
